app/components/index-select/component.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import ENV from '../../config/environment';
+
+
+export default Ember.Component.extend({
+	didRender(){
```

Comment:
I think modifying css after a render causes a page reflow. Would binding the style like this work?http://jsfiddle.net/tomwhatmore/rearT/1/ Instead of a string one could use a computed